### PR TITLE
Add expand business unit to apiv2 applications

### PR DESCRIPTION
**Pull request by emintham**
Refs #SDE-2699

### CODE REVIEW

File: source/includes/applications.md [hunk 1]
```diff
@@ -44,6 +44,57 @@ search        | Filter applications by performing a textual search on name.
...
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "results": [{
+        "id": 3,
+        "business_unit": {
```
Comment (funkaoshi) [inline source/includes/applications.md]: Maybe this is a better discussion for the other MR, but I wonder if we should have a smaller inline serializer that only sends back a smaller set of information: id, slug, name? Though maybe we can worry more about whether our endpoints are huge later.

**Comment by funkaoshi** [comment]
+1

**Comment by sibartlett** [comment]
This PR needs a milestone

**Comment by emintham** [comment]
Done.

**Pull request merged**